aaelf32/aaelf32.rst

```diff
@@ -0,0 +1,3289 @@
+  +=============================+============+==================================================================================+
+  | PT_ARM_ARCHEXT_PROF_NONE    | 0x00000000 | The architecture has no profile variants, or the image has no profile-specific   |
+  |                             |            | constraints                                                                      |
+  +-----------------------------+------------+----------------------------------------------------------------------------------+
+  | PT_ARM_ARCHEXT_PROF_ARM     | 0x00410000 |                                                                                  |
+  |                             |            |                                                                                  |
+  |                             | (‘A’<<16)  | The executable file requires the Application profile                             |
+  +-----------------------------+------------+----------------------------------------------------------------------------------+
+  | PT_ARM_ARCHEXT_PROF_RT      | 0x00520000 |                                                                                  |
+  |                             |            |                                                                                  |
+  |                             | (‘R’<<16)  | The executable file requires the Real-Time profile                               |
+  +-----------------------------+------------+----------------------------------------------------------------------------------+
+  | PT_ARM_ARCHEXT_PROF_MC      | 0x004D0000 |                                                                                  |
+  |                             |            |                                                                                  |
+  |                             | (‘M’<<16)  | The executable file requires the Microcontroller profile                         |
+  +-----------------------------+------------+----------------------------------------------------------------------------------+
+  | PT_ARM_ARCHEXT_PROF_CLASSIC | 0x00530000 |                                                                                  |
+  |                             |            |                                                                                  |
+  |                             | (‘S’<<16)  | The executable file requires the ‘classic’ (‘A’ or ‘R’ profile) exception model. |
+  +-----------------------------+------------+----------------------------------------------------------------------------------+
+
+The following table defines the values that specify the minimum architecture
+version needed by this executable file. These values are identical to those of
+the Tag_CPU_arch attribute used in the attributes section [Addenda32_] of a
+relocatable file.
+
+.. _Architecture version compatibility data:
+
+.. table:: Architecture version compatibility data
+  :widths: 37 13 50
```

Comment:
as described above on the version history table, please use a class to specify the width